packages/php-wasm/universal/src/lib/urls.ts

```diff
@@ -59,3 +59,15 @@ export function ensurePathPrefix(path: string, prefix: string): string {
 	}
 	return prefix + path;
 }
+
+/**
+ * Checks if the given URL is an absolute URL.
+ *
+ * Absolute URLs start with `http://` or `https://`.
+ *
+ * @param  url - The URL string to check.
+ * @returns True if the URL is absolute, false otherwise.
+ */
+export function isAbsoluteUrl(url: string): boolean {
+	return url.startsWith('http://') || url.startsWith('https://');
```

Comment:
I think we can just use [`URL.canParse()`](https://developer.mozilla.org/en-US/docs/Web/API/URL/canParse_static) instead of this function. As a side benefit, it is a more complete check for full URLs.

Reply:
TIL `URL.canParse()`

I updated the helper function in f1e7092, but I'm wondering if `isAbsoluteUrl` is even needed?

Reply:
Unfortunately we can't use `URL.canParse()`, [it requires Node `19.9`](https://developer.mozilla.org/en-US/docs/Web/API/URL/canParse_static#browser_compatibility) and [we use `18.18`](https://github.com/WordPress/wordpress-playground/blob/cb55349ec346574f2975760e2c92a78632f7e5a9/.nvmrc#L1).

Reply:
Any polyfills available?

Reply:
Yes, MDN links to a [core-js polyfill](https://github.com/zloirock/core-js/blob/master/packages/core-js/modules/web.url.can-parse.js). I can add it to node-polyfills package, but it looks like we might have a different issue here.

I initially check on MDN for URL.canParse support, but on Node.js, [they also list `v18.17` ](https://nodejs.org/docs/latest-v21.x/api/url.html#urlcanparseinput-base). 
When I tested `URL.canParse` in a Node console with node 18.18 it worked, so it looks to me like our unit tests don't use Node 18.18. 👀

Reply:
I checked the Node version using `execSync('node --version').toString().trim()` and it is 18.18, so URL.canParse should work.

Reply:
I'm out of ideas. I made sure unit tests use Node 18.18 and that it supports `URL.canParse`.
@adamziel @brandonpayton do you have any suggestions what I could take a look at next or why `URL.canParse` isn't supported?

Reply:
Thanks! I added a polyfill for URL.canParse [4430e2b](https://github.com/WordPress/wordpress-playground/pull/1978/commits/4430e2b4814e7dd34b4305a0196e4a7d321a903a)

Reply:
Lovely, thank you